pkg/apis/v1alpha1/register.go

```diff
@@ -82,21 +86,14 @@ var (
 	LabelInstanceMemory                       = LabelDomain + "/instance-memory"
 	LabelInstanceNetworkBandwidth             = LabelDomain + "/instance-network-bandwidth"
 	LabelInstancePods                         = LabelDomain + "/instance-pods"
-	// TODO: will deprecate at v1beta1, remove at v1
-	LabelInstanceGPUName = LabelDomain + "/instance-gpu-name"
-	// TODO: will deprecate at v1beta1, remove at v1
-	LabelInstanceGPUManufacturer = LabelDomain + "/instance-gpu-manufacturer"
-	// TODO: will deprecate at v1beta1, remove at v1
-	LabelInstanceGPUCount = LabelDomain + "/instance-gpu-count"
-	// TODO: will deprecate at v1beta1, remove at v1
-	LabelInstanceGPUMemory               = LabelDomain + "/instance-gpu-memory"
-	LabelInstanceAMIID                   = LabelDomain + "/instance-ami-id"
-	LabelInstanceAcceleratorName         = LabelDomain + "/instance-accelerator-name"
-	LabelInstanceAcceleratorManufacturer = LabelDomain + "/instance-accelerator-manufacturer"
-	LabelInstanceAcceleratorCount        = LabelDomain + "/instance-accelerator-count"
-	LabelInstanceAcceleratorMemory       = LabelDomain + "/instance-accelerator-memory"
-
-	InterruptionInfrastructureFinalizer = Group + "/interruption-infrastructure"
```

Comment:
Do we want to remove this?

Reply:
I'm not sure which thing you are refering to here but the only thing that's being removed is the `LabelInstanceAcceleratorMemory` and the `InterruptionInfrastructureFinalizer` which is an artifact of code changes that are no longer valid